playbooks/preflight-check.yml

```diff
@@ -138,3 +138,21 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1371 detect if enough space for rootfs backup
+- name: Check space for rootfs backup on infras
+  hosts: "os-infra_hosts"
+  user: root
+  tasks:
+    - name: Indentify the space used in /
+      shell: |
+        df -h / | tail -n 1 | awk '{print $5}' | sed 's/%//'
+      register: rootfs_space_used
+
+    - name: Check space if enough for rootfs backup
+      fail:
+        msg: 
+          - "Error: There wasn't at least 50% space available on infra nodes for rootfs backup."
+          - "You can ignore this check by skipping tag 'rootfs_backup_space_check' on 'preflight-check.yml' manually. Then rerun ubuntu14-leapfrog.sh."
+      when: rootfs_space_used.stdout > '50'
+      tags: rootfs_backup_space_check
```

Comment:
Can we move this under the main play and make it a list like the others in the file?

    tags:
      - rootfs_backup_space_check

If they skip the tag, they'll probably just want to skip the whole play.

Reply:
Yes just move the tag up to the main play